test/vaadin-list-mixin.html

```diff
@@ -523,17 +523,50 @@
           list.multiple = true;
           list.items[1].click();
           expect(list.selectedValues).to.eql([1]);
+          expect(list.items[1].selected).to.be.true;
           list.items[3].click();
           expect(list.selectedValues).to.eql([1, 3]);
+          expect(list.items[1].selected).to.be.true;
         });
 
         it('should remove item from the selectedValues when clicking', () => {
           list.multiple = true;
           list.selectedValues = [1, 3];
           list.items[1].click();
           expect(list.selectedValues).to.eql([3]);
+          expect(list.items[1].selected).to.be.false;
           list.items[3].click();
           expect(list.selectedValues).to.eql([]);
+          expect(list.items[3].selected).to.be.false;
+        });
+
+        it('should fire one selected-values-changed event', () => {
```

Comment:
Should also test that `event.detail.value` contains the array (instead of `splices` or `length` like when mutating the array)

Reply:
Updated to check the value to be array with clicked items index.